lib/web_ui/dev/chrome_installer.dart

```diff
@@ -156,9 +155,9 @@ class ChromeInstaller {
       versionDir.createSync(recursive: true);
     }
 
-    print('INFO: Starting Chrome download.');
-
     final String url = PlatformBinding.instance.getChromeDownloadUrl(version);
```

Comment:
Reading this code, it looks like we always re-download Chrome even if LUCI has provided us with a Chrome from CIPD. In fact, in the code above, we check to make sure that LUCI provided a Chrome installation, but here we download it again anyway.

Reply:
Good point. We actually shouldn't enter this method when running on LUCI because the call is predicated on `installer.isInstalled` above:

```dart
if (installer.isInstalled) {
      infoLog.writeln(
          'Installation was skipped because Chrome version ${installer.version} is already installed.');
    }
} else {
  // calls this method
}
```

I'm going to assert here that we're not running on LUCI and see what happens.